resources/arduino_files/shared/interval_timer.cpp

```diff
@@ -0,0 +1,41 @@
+#include "interval_timer.h"
```

Comment:
Seems like this is something that should already exist publicly.

Reply:
It does, and I stole from it when I wrote the original code last summer. But it is part of a much larger library.